project/Settings.scala

```diff
@@ -93,7 +93,7 @@ object Settings {
         case _ => publishTo.value
       }
     },
-
+    excludeDependencies += "log4j" % "log4j",
```

Comment:
@metasim thx for the PR! 

As far as I know, `excludeDependencies`would not be reflected in the published artifacts (i.e. see https://repo1.maven.org/maven2/org/locationtech/geotrellis/geotrellis-geotools_2.12/3.6.1/geotrellis-geotools_2.12-3.6.1.pom), could you double check the behavior / switch to using `ExclusionRules` instead?